addons/account/models/account_journal_dashboard.py

```diff
@@ -884,6 +898,31 @@ def show_sequence_holes(self):
             }
         }
 
+    def show_unhashed_entries(self):
+        self.ensure_one()
+        chains_to_hash = self.env['account.move'].search([
+            ('restrict_mode_hash_table', '=', True),
+            ('inalterable_hash', '=', False),
+            ('journal_id', '=', self.id),
+            ('date', '>', self.company_id._get_user_fiscal_lock_date())
+        ])._get_chains_to_hash(force_hash=True, raise_if_gap=False, raise_if_no_document=False, include_pre_last_hash=True)
```

Comment:
It's intended:

INV/4 not hashed (v3 ou resesquence), INV/5/6/7/8 hashed
-> Do not show warning

INV/4 not hashed (v3 ou resesquence), INV/5/6/7/8 hashed, INV/9 not yet
-> Show warning on dashboard
-> open list view of INV/4, INV/9 (or form view if only one)
-> form INV/4 button_hash = error
-> form INV/9 button_hash = OK

Reply:
But why show it if it raises an error when you try to hash it? :smile:

Reply:
We show it because the user may still want to resequence it (IIRC @tsb-odoo) but we don't want to pollute the dashboard with a warning for non-hashed moves that were previous to v4